flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/mysql/DateToStringConverter.java

```diff
@@ -47,26 +48,42 @@ public class DateToStringConverter implements CustomConverter<SchemaBuilder, Rel
     public static final Properties DEFAULT_PROPS = new Properties();
 
     static {
-        DEFAULT_PROPS.setProperty("converters", "date");
+        DEFAULT_PROPS.setProperty(DatabaseSyncConfig.CONVERTERS, DatabaseSyncConfig.DATE);
         DEFAULT_PROPS.setProperty(
-                "date.type", "org.apache.doris.flink.tools.cdc.mysql.DateToStringConverter");
-        DEFAULT_PROPS.setProperty("date.format.date", "yyyy-MM-dd");
-        DEFAULT_PROPS.setProperty("date.format.datetime", "yyyy-MM-dd HH:mm:ss.SSSSSS");
-        DEFAULT_PROPS.setProperty("date.format.timestamp", "yyyy-MM-dd HH:mm:ss.SSSSSS");
-        DEFAULT_PROPS.setProperty("date.format.timestamp.zone", "UTC+8");
+                DatabaseSyncConfig.DATE_TYPE,
+                "org.apache.doris.flink.tools.cdc.mysql.DateToStringConverter");
+        DEFAULT_PROPS.setProperty(
+                DatabaseSyncConfig.DATE_FORMAT_DATE, DatabaseSyncConfig.YEAR_MONTH_DAY_FORMAT);
+        DEFAULT_PROPS.setProperty(
+                DatabaseSyncConfig.DATE_FORMAT_DATETIME, DatabaseSyncConfig.DATETIME_MICRO_FORMAT);
+        DEFAULT_PROPS.setProperty(
+                DatabaseSyncConfig.DATE_FORMAT_TIMESTAMP, DatabaseSyncConfig.DATETIME_MICRO_FORMAT);
+        DEFAULT_PROPS.setProperty(
+                DatabaseSyncConfig.DATE_FORMAT_TIMESTAMP_ZONE, DatabaseSyncConfig.TIME_ZONE_UTC_8);
```

Comment:
If it is only used here, I think it is better to define it inside this convert class

Reply:
Each source corresponds to a DateConverter, which basically contains these parameters.